Content.Shared/Vehicle/Components/VehicleComponent.cs

```diff
@@ -0,0 +1,98 @@
+using Content.Shared.Actions.ActionTypes;
+using Content.Shared.Sound;
+using Content.Shared.Containers.ItemSlots;
+using Robust.Shared.Audio;
+using Robust.Shared.Utility;
+using Content.Shared.Whitelist;
+
+namespace Content.Shared.Vehicle.Components
+{
+    /// <summary>
+    /// This is particularly for vehicles that use
+    /// buckle. Stuff like clown cars may need a different
+    /// component at some point.
+    /// All vehicles should have Physics, Strap, and SharedPlayerInputMover components.
+    /// </summary>
+    [RegisterComponent]
+    public sealed class VehicleComponent : Component
+    {
+        /// <summary>
+        /// Whether someone is currently riding the vehicle
+        /// </summary
+        public bool HasRider = false;
+
+        /// <summary>
+        /// The entity currently riding the vehicle.
+        /// </summary>
+        [ViewVariables]
+        public EntityUid? Rider;
+
+        /// <summary>
+        /// Whether the vehicle should treat north as it's unique direction in its visualizer
+        /// </summary>
+        [DataField("northOnly")]
+        public bool NorthOnly = false;
+
+        /// <summary>
+        /// What the y buckle offset should be in north / south
+        /// </summary>
+        [DataField("northOverride")]
+        public float NorthOverride = 0f;
+
+        /// <summary>
+        /// What the y buckle offset should be in north / south
+        /// </summary>
+        [DataField("southOverride")]
+        public float SouthOverride = 0f;
+
+        /// <summary>
+        /// The base offset for the vehicle (when facing east)
+        /// </summary>
+        public Vector2 BaseBuckleOffset = Vector2.Zero;
+
+        /// <summary>
+        /// The sound that the horn makes
+        /// </summary>
+        [DataField("hornSound")]
+        public SoundSpecifier? HornSound = new SoundPathSpecifier("/Audio/Effects/Vehicle/carhorn.ogg");
+
+        /// <summary>
+        /// Whether the horn is a siren or not.
+        /// </summary>
+        [DataField("hornIsSiren")]
+        public bool HornIsSiren = false;
+
+        /// <summary>
+        /// If this vehicle has a siren currently playing.
+        /// </summary>
+        public bool SirenPlaying = false;
```

Comment:
i assume the distinction here is "plays a sound once" vs "looping audio", maybe make that clearer by naming these `HornIsLooping`, etc